tests/TestCase/Routing/RouterTest.php

```diff
@@ -3422,6 +3422,53 @@ public function testCreateRouteBuilder()
         $this->assertSame(['json'], $builder->getExtensions());
     }
 
+    /**
+     * testShortStringSyntax
+     *
+     * @return void
+     */
+    public function testShortStringSyntax()
+    {
+        Router::connect('/admin/articles/view', 'Admin\Articles::view');
+        $result = Router::parseRequest($this->makeRequest('/admin/articles/view', 'GET'));
+        $expected = [
+            'pass' => [],
+            'prefix' => 'Admin',
+            'controller' => 'Articles',
+            'view' => 'view',
+            'action' => 'index',
+            'plugin' => null,
+            '_matchedRoute' => '/admin/articles/view'
+
+        ];
+        $this->assertEquals($result, $expected);
+
+        Router::connect('/admin/blog/articles/view', 'Blog.Admin\Articles::view');
```

Comment:
What about nested prefixes? Those have caused us troubles in other places.

Reply:
Or all tokens separated by \ except the last one are the prefix, and the last one is the controller name.

Reply:
@markstory like that?

![image](https://user-images.githubusercontent.com/162789/35798348-cfbe787a-0a62-11e8-96db-76e48408215e.png)

Reply:
Is `Foo\Bar\Admin` the prefix? Or the plugin name there?

Reply:
Prefix? Just specific exactly how you would like to have it implemented? :) Can you just commit a test case for the syntax you want and I'll take care of the rest.